src/OrbitGl/MultivariateTimeSeries.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitGl/MultivariateTimeSeries.h"
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_gl {
+
+MultivariateTimeSeries::MultivariateTimeSeries(absl::Span<const std::string> series_names,
+                                               uint8_t value_decimal_digits, std::string value_unit)
+    : series_names_(series_names.begin(), series_names.end()),
```

Comment:
Instead of using an absl::Span` for the series_names it would be more idiomatic to use a `std::vector<std::string>` and then move it into `series_names_`.

Rule of thumb: Only use `Span<const T>` as a replacement for `const std::vector<T>&` - otherwise getting the series names by value und moving is the better option.